orchestrator-salt/src/main/resources/salt/salt/disks/mount/scripts/mount-disks.j2

```diff
@@ -122,31 +122,19 @@ mount_one() {
       return $((return_value))
 }
 
-mount_tempstorage_azure() {
+demount_tempstorage_azure() {
       local return_value=0
-      local success=0
       local waagent_conf="/etc/waagent.conf"
       local original_path="/mnt/resource"
-      local path="/hadoopfs/ephfs1"
       local device=$(df | grep $original_path | awk '{print $1}')
 
-      log $LOG_FILE unmounting $device temporary storage device from path $original_path
+      log $LOG_FILE demounting $device temporary storage device from path $original_path
       umount $original_path >> $LOG_FILE 2>&1
       if [ ! $? -eq 0 ]; then
-        log $LOG_FILE error unmounting temporary storage device on $original_path
+        log $LOG_FILE error demounting temporary storage device on $original_path
         return_value=1
       fi
-      log $LOG_FILE result of unmounting $original_path: $?
-      log $LOG_FILE mounting $device temporary storage device on path $path
-      mkdir $path >> $LOG_FILE 2>&1
-      log $LOG_FILE result of creating mount directory $path: $?
-      mount -t $FS_TYPE -o defaults,noatime,nofail $device $path >> $LOG_FILE 2>&1
-      if [ ! $? -eq 0 ]; then
-        log $LOG_FILE error mounting temporary storage device on $path
-        return_value=1
-      fi
-      log $LOG_FILE result of mounting $path: $?
-      chmod 777 $path >> $LOG_FILE 2>&1
+      log $LOG_FILE result of demounting $original_path: $?
```

Comment:
Does it make sense to proceed with the next steps if `umount` fails for any reason?

Reply:
yes, because the next part takes care of disabling the default azure temp disk management that should run independently of the outcome umount.